### ISSUE TITLE
boolean serialised with uppercase first letter

### ISSUE THREAD

**Issue by maurosilber** [dapper91/pydantic-xml#126]
I'm not sure if this is an invalid XML, but in https://www.w3.org/TR/xmlschema-2/#boolean says that the valid values are `{true, false, 1, 0}`:

```python
from pydantic_xml import BaseXmlModel, attr


class Model(BaseXmlModel):
    x: bool = attr()


print(
    Model
    .from_xml(b'<Model x="true" />')
    .to_xml()
    .decode()
)
```
```xml
<Model x="True"/>
```

**Comment by maurosilber** [comment]
Yes, I did that, thanks.

But shoudn't it by default encode the Python `True` as the string `"true"`, which is what the XML specification says?

**Comment by dapper91** [comment]
@maurosilber  Yes, that sounds reasonable, fixed it in version 2.3.0.
